package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "module_4",
```

Comment:
I get a linting error here: ``` 2:9  error  Parsing error: Unexpected token :```
I don't understand why I'm getting this error, because this is the standard package.json format, AFAIK. Help?